lab1/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="app.css">
+	<title>World of Coins</title>	
+</head>
+<body>
+	<h1>World of Coins</h1>
+	<textarea class="terminal">Data.................................</textarea>
```

Comment:
I'd refrain from using `textarea` for displaying readonly data purpose. It's not meant for it. Textarea usually implies the data is used for some kind forms.